ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CCDCase.java

```diff
@@ -40,6 +40,7 @@
     private String interestReason;
     private BigDecimal interestSpecificDailyAmount;
     private CCDInterestDateType interestDateType;
+    private String currentInterestAmount;
```

Comment:
As in CCD this is going to be of type `MoneyGBP`. Please make sure you populate pence instead of pounds in here. A MoneyGBP value is a string representing a monetary value in pence.

Reply:
Oof! Thanks for the tip. Didn't realise that was the case, I'll double check the logic